client/src/components/profileInfoEdit/ProfileDeleteModal.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Delete Alert Modal component display a modal and ask for confirmation when the delete button is clicked
+ */
```

Comment:
This file has almost the same code with ProfileClearModal.js, can you refactor it so that it's in the same file?